web/src/components/hooks/useFeatures.ts

```diff
@@ -17,13 +22,16 @@ const noFeatures: features = {
   incidents: false,
 };
 
-export const useFeatures = () => {
+// monitoring-console-plugin proxy via. cluster observability operator
+// https://github.com/rhobs/observability-operator/blob/28ac1ba9e179d25cae60e8223bcf61816e23a311/pkg/controllers/uiplugin/monitoring.go#L44
+const MCP_PROXY_PATH = '/api/proxy/plugin/monitoring-console-plugin/backend';
+const featuresEndpoint = `${MCP_PROXY_PATH}/features`;
+
+export const useFeaturesPolling = (pollDelay: POLL_DELAY = URL_POLL_DEFAULT_DELAY) => {
```

Comment:
I don't see any usage of this function. Should we keep it?

Reply:
I removed it in the latest push.